src/components/Slider/Slider.js

```diff
@@ -70,9 +70,15 @@ function Slider(props) {
     if (!disabled) {
       evt.persist();
       evt.imaginaryTarget = _inputRef;
-
-      setValue(evt.target.value);
-      onChange(parseFloat(evt.target.value), evt);
+      if(evt.target.value > max){
```

Comment:
Does this also work with `min`, for example with a range from `20-50` and somebody enters `10`?